packer/nomad-server/image.pkr.hcl

```diff
@@ -0,0 +1,163 @@
+packer {
```

Comment:
All the comments for the other template apply for this one.

What is the actual difference between these? They look the same. Anything you can do with the overall structure of these files to help clarify this will be useful.

Reply:
merged em